<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>949</width>
    <height>662</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QSlider" name="verticalSliderR">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="maximum">
              <number>255</number>
             </property>
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="spinBoxR">
             <property name="font">
              <font>
               <pointsize>20</pointsize>
              </font>
             </property>
             <property name="maximum">
              <number>255</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QSlider" name="verticalSliderG">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="maximum">
              <number>255</number>
             </property>
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="spinBoxG">
             <property name="font">
              <font>
               <pointsize>20</pointsize>
              </font>
             </property>
             <property name="maximum">
              <number>255</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="QSlider" name="verticalSliderB">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="maximum">
              <number>255</number>
             </property>
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="spinBoxB">
             <property name="font">
              <font>
               <pointsize>20</pointsize>
              </font>
             </property>
             <property name="maximum">
              <number>255</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="labelRGB">
         <property name="font">
          <font>
           <pointsize>50</pointsize>
          </font>
         </property>
         <property name="autoFillBackground">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>RGB</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="0" column="1">
      <layout class="QVBoxLayout" name="verticalLayout_7">
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Color Mode</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <item>
           <widget class="QRadioButton" name="radioButtonRGB">
            <property name="text">
             <string>RGB Mode</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="radioButtonCMY">
            <property name="text">
             <string>CMY Mode</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QListView" name="listColor"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>verticalSliderR</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBoxR</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>121</x>
     <y>222</y>
    </hint>
    <hint type="destinationlabel">
     <x>113</x>
     <y>453</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBoxR</sender>
   <signal>valueChanged(int)</signal>
   <receiver>verticalSliderR</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>155</x>
     <y>464</y>
    </hint>
    <hint type="destinationlabel">
     <x>173</x>
     <y>349</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBoxG</sender>
   <signal>valueChanged(int)</signal>
   <receiver>verticalSliderG</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>372</x>
     <y>442</y>
    </hint>
    <hint type="destinationlabel">
     <x>384</x>
     <y>361</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>verticalSliderG</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBoxG</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>336</x>
     <y>247</y>
    </hint>
    <hint type="destinationlabel">
     <x>312</x>
     <y>459</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>verticalSliderB</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBoxB</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>566</x>
     <y>309</y>
    </hint>
    <hint type="destinationlabel">
     <x>545</x>
     <y>461</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBoxB</sender>
   <signal>valueChanged(int)</signal>
   <receiver>verticalSliderB</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>480</x>
     <y>464</y>
    </hint>
    <hint type="destinationlabel">
     <x>491</x>
     <y>382</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
